portfolio/src/main/webapp/index.html

```diff
@@ -7,17 +7,16 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div class="header">
+    <ul class="header">
       <img src="images/Profile_photo_Jess.jpg" alt="Profile photo of Jess" height="250">
       <h1>Jess's Portfolio</h1>
-      <strong>
-        <a href="#">About Me</a>
-        <a href="#">Projects</a>
-        <a href="#">Experience</a>
-        <a href="#">Hobbies</a>
-        <a href="#">Socials & Contact</a>
-      </strong>
-    </div>
+      <b>
```

Comment:
nit: instead of using a <b> tag, just use CSS

```css
.header li {
  /* ... */
  font-weight: bold;
}
```

Reply:
Makes sense, I'll make the change.